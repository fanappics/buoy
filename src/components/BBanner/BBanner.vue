<template>
  <header role="banner">
    <!-- <b-nav></b-nav> -->
    <slot name="nav"></slot>
  </header>
</template>

<script>
import BNav from '../BNav'

export default {
  name: 'b-banner',
  component: [BNav],
  props: {

  },
  mounted () {
    if (this.$el.parentElement !== document.body)
      console.error('Parent element for <b-name> must be body.')
  }
}
</script>

<style lang="stylus" scoped>

</style>
