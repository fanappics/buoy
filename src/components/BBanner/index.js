import BBanner from './BBanner.vue'

/* istanbul ignore next */
BBanner.install = function install (Vue) {
  Vue.component(BBanner.name, BBanner)
}

export default BBanner
