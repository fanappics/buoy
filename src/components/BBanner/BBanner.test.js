import { mount, shallow } from 'vue-test-utils'
import BBanner from './BBanner'

describe('BBanner', () => {
  it('renders a button', () => {
    const wrapper = mount(BBanner)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
