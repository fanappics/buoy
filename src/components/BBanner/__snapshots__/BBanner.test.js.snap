// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BBanner renders a button 1`] = `<header role="banner"></header>`;
